projects/tech-meetings/2025/2025-06-24.md

```diff
@@ -0,0 +1,42 @@
+## Agenda
+
+- Review Actions from last meeting see also https://github.com/orgs/datafoodconsortium/projects/5/views/1
+- SHACKL SHAPE vs UML
+- Versionning
+- WebID
+- Define next meeting date: July 8th 4PM CET
+
+## Participants
+
+- @RaggedStaff
+- @jgaehring
+- @lecoqlibre
+- @simonLouvet
+- @RachL
+
+## SHCL shape (https://github.com/datafoodconsortium/connector-codegen/issues/34)
+
+Maxime is proposing to use shacl shape instead of the current UML model.
+
+We've discussed some pros and cons:
+
+- Pros : The output is easier to read and we can replace Aceleo which isn't has easy to handle as we first thought
+- Cons: we need to create and maintain a code generator. This has been evaluated at 5 days for which we don't have budget yet
+
+Actions: everyone should take a look at the proposal so we can discussed it next week?
+
+## Communicate ersion of DFC Standard in response header (https://github.com/datafoodconsortium/standard/issues/54)
+
+While we wait for webid to be available, we need to be able to know which version is a platform using. we could use the context, but the context can be linked to what we've published on the standard website. In that case using the header can be a workaround in the meantime.
+
+## WebID and profile (https://github.com/datafoodconsortium/standard/issues/10)
+
+We've clarified that we can use the index to search for an enterprise without using the user. Using the user is only an additionnal option.
+
+Action: Maxime is going to create a change request in gitbook for everyone to reiew
+
+## Reminders
+
+- Modularizing our ontology and version control: this needs an update from the ontology circle
```

Comment:
It might be a good point to discuss version control? I can't recall the detail but I'll try & find it... anyone have a link?

Reply:
the only one I have is here: https://github.com/orgs/datafoodconsortium/discussions/19